prisma/schema.prisma

```diff
@@ -56,6 +56,8 @@ model achievement_template {
   actionType                achievement_action_type_enum?
   // Tiered: If a tiered achievement has been achieved, the text changes
   achievedText              String?                       @db.VarChar
+  progressDescription       String?                       @db.VarChar
+  streakProgress            String?                       @db.VarChar
```

Comment:
I will add an explanation in the documentation

Reply:
Ok no the information is not that much so I'll answer you here. Streak description is the description on a streak card, that no other module has.

Reply:
Am I right that:
Description: will be shown in the big card
streakProgress: will be shown in the small card in case the current streak is ongoing
progressDescription: will be shown in the small card in case you broke the last streak, but you are on the way to a new one
?